package.json

```diff
@@ -17,39 +17,41 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "bluebird": "3.4.6",
-    "body-parser": "1.15.2",
-    "express": "4.14.0",
-    "express-session": "1.14.0",
-    "express-winston": "2.0.0",
-    "lodash": "4.17.4",
-    "nconf": "0.8.4",
-    "request": "2.74.0",
+    "@google-cloud/connect-datastore": "^1.0.0",
+    "@google-cloud/datastore": "0.6.0",
+    "@google-cloud/debug-agent": "3.0.0",
+    "@google-cloud/trace-agent": "3.3.1",
+    "bluebird": "3.5.3",
+    "body-parser": "1.18.3",
+    "express": "4.16.4",
+    "express-session": "1.15.6",
+    "express-winston": "2.6.0",
+    "lodash": "4.17.11",
+    "nconf": "0.10.0",
+    "request": "2.88.0",
     "winston": "2.2.0",
     "yamljs": "0.2.8"
   },
   "devDependencies": {
     "@babel/core": "7.0.0",
     "@babel/preset-env": "7.0.0",
     "@babel/register": "7.0.0",
-    "@google-cloud/debug-agent": "2.6.0",
-    "@google-cloud/trace-agent": "3.1.1",
     "ava": "1.0.0-rc.2",
     "babel-eslint": "8",
     "del": "2.2.2",
     "eslint": "4.x",
     "eslint-plugin-ava": "5.1.1",
     "gulp": "3.9.1",
     "gulp-ava": "0.19.0",
-    "gulp-babel": "8.0.0-beta.2",
+    "gulp-babel": "8.0.0",
     "gulp-batch": "1.0.5",
     "gulp-eslint": "3.0.1",
     "gulp-if": "2.0.2",
     "gulp-load-plugins": "1.5.0",
     "gulp-watch": "4.3.11",
     "lazypipe": "1.0.1",
     "minimist": "1.2.0",
-    "nodemon": "^1.18.6",
+    "nodemon": "1.18.6",
```

Comment:
IIRC, we upgraded to `1.18.7` for `ampproject/amphtml`

Reply:
goot catch.